power-client.yml

```diff
@@ -6,6 +6,7 @@
     - name: command
       prompt: "Befehl eingeben: (status, poweroff, poweron)"
       default: "status"
+      private: no
```

Comment:
In Ansible 2.9.6 ist der dafault: private:yes -> D.H, es ist wie ein Kennwort ohne Rückemdlung was man eingibt